src/util/is-cjs.ts

```diff
@@ -0,0 +1 @@
+export const isCjs = false;
```

Comment:
how does this determine, whether it's esm or cjs?

Reply:
By replacing it with [these contents](https://github.com/oceanroleplay/discord.ts/pull/234/files/de9799a6e3357986c64ca8129836c2d924f8e698#diff-fe0e8c4367ef7eb68d0d2d40571c1f44b1c45a3fd4b0233bfa4121134e9a975bR1) via [this script](https://github.com/oceanroleplay/discord.ts/pull/234/files/de9799a6e3357986c64ca8129836c2d924f8e698#diff-5e2affc52c1ef28125aef46a65387219d79f6fde1c1bb7eaffbb1b486e9e3419R16).

Reply:
I appreciate the workaround this with postfix, it will work and this PR is helpful. But unfortunately, I have now planned to remove ``classes`` field from client. Which makes this changes invalid for merge.